src/pages/TermsOfService.js

```diff
@@ -66,18 +61,7 @@ const TermsOfServicePage = () => {
   }
 
   const continueButton = () => {
-    try {
```

Comment:
I suspect this was originally copied from the `accept` function. While `accept` contains an API request, there's nothing here that requires the busy state or error handling.